docs/basics/object-spawning.md

```diff
@@ -228,7 +228,7 @@ public class SinglePooledDynamicSpawner : NetworkBehaviour, INetworkPrefabInstan
     {
         // We register our network Prefab and this NetworkBehaviour that implements the
         // INetworkPrefabInstanceHandler interface with the Prefab handler
-        NetworkManager.PrefabHandler.AddHandler(PrefabToSpawn, this);
+        ScriptableObject.PrefabHandler.AddHandler(PrefabToSpawn, this);
```

Comment:
This is unchanged. The NetworkManager APIs all stayed as they were. The only thing that's changed is that there are now two ways to add a prefab:

`NetworkManager.AddNetworkPrefab()` will add a prefab to the individual network manager and must be called after the network manager's initialized or it'll git overwritten during initialization.

Or `NetworkManager.NetworkConfig.Prefabs.NetworkPrefabsLists[i].Add()` which will add it to the ScriptableObject (at runtime, but will not persist on disk) and will affect all NetworkManagers that point at that NetworkPrefabsList in the local process.

But all the existing code examples should remain unchanged.